### PR TITLE
Switch to styled-components

### PR DESCRIPTION
**Pull request by mxstbr**
:trollface: ?

(yes it works)

### CODE REVIEW

File: src/components/App/index.js [hunk 1]
```diff
@@ -145,7 +145,9 @@ function Home() {
             <S.Link href="mailto:contact@twizzy.app">Contact</S.Link>
             <S.Link href="privacy.html">Privacy</S.Link>
             <S.Link href="disclaimer.html">Disclaimer</S.Link>
-            <S.Link target="_blank" rel="noopener" href="https://github.com/kitze/twizzy-landing">View source</S.Link>
+            <S.Link target="_blank" rel="noopener" href="https://github.com/kitze/twizzy-landing">
+              View source
+            </S.Link>
```
Comment (mxstbr) [inline src/components/App/index.js]: Prettier auto-formatted

File: config-overrides.js [hunk 2]
```diff
@@ -1,12 +1,8 @@
-const {
-  override,
-  disableEsLint,
-  addBabelPlugin
-} = require('customize-cra');
+const { override, disableEsLint, addBabelPlugin } = require('customize-cra');
```
Comment (mxstbr) [inline config-overrides.js]: Prettier auto-formatted

**Comment by mxstbr** [comment]
Yep! We added support for that recently. Object styles are now a first-class citizen!

**Comment by kitze** [comment]
Merged. Dammit @mxstbr your marketing game is 💯️

I went from aphrodite -> styled-components -> glamorous -> emotion -> styled-components.

**Comment by Andarist** [comment]
This is kinda a weird PR to me - maybe because I'm more involved in developing emotion but I love both projects.

It's weird to me because nothing was really added, no argument was given in favour of the change. Seems like a marketing/power grab move to me, which doesn't sound nice. I don't want to accuse anyone of anything, but this is really strange to me. 

That being said ofc anyone is free to use any solution they want 😉

**Comment by mxstbr** [comment]
Definitely meant as a joke :trollface: (I didn't think @kitze would actually merge this PR)

**Pull request merged**